### PR TITLE
Add 0.203 release notes

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.179.rst [hunk 1]
```diff
@@ -5,9 +5,9 @@ Release 0.179
 General Changes
 ---------------
 
-* Fix issue which could cause incorrect results when processing dictionary encoded data. If the expression
-  can fail on bad input, the results from filtered-out rows containing bad input may be included in the query
-  output. See `#8262 <https://github.com/prestodb/presto/issues/8262>`_ for more details.
+* Fix issue which could cause incorrect results when processing dictionary encoded data.
+  If the expression can fail on bad input, the results from filtered-out rows containing
+  bad input may be included in the query output (:issue:`8262`).
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.179.rst]: nice

File: presto-docs/src/main/sphinx/release/release-0.203.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.203
+=============
+
+General Changes
+---------------
+
+* Fix spurious duplicate key errors from :func:`map`.
+* Fix planning failure when correlated subqueries containing a ``LIMIT``
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.203.rst]: I thnink singular will be easier to read "Fix planning failure when a correlated subquery containing a ``LIMIT`` clause is used within ``EXISTS`` (:issue:`10696`).

File: presto-docs/src/main/sphinx/release/release-0.203.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  clause are used within ``EXISTS`` (:issue:`10696`).
+* Fix out of memory error caused by missing pushback checks in data exchanges.
+* Fix execution failure for queries with cross joins when using bucketed execution.
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.203.rst]: maybe "Fix execution failure for queries containing a cross join when using bucketed execution."?

File: presto-docs/src/main/sphinx/release/release-0.203.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+* Fix complex type handling in the optimized Parquet reader. Previously, null values,
+  optional fields, and Parquet backward compatibility rules were not handled correctly.
+* Fix an issue that could cause the optimized ORC writer to fail with a ``LazyBlock`` error.
+* Improve error message for max open writers.
+
+Thrift Connector Changes
+------------------------
+
+* Fix retry of requests when the remote Thrift server indicates that the
+  error is retryable.
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.203.rst]: should we have a blankline at the end of the file?

Comment (electrum) [reply presto-docs/src/main/sphinx/release/release-0.203.rst]: No, as with other source files, it ends in a newline, but not an extra blank line.

**Pull request closed without merging**